Extension/src/LanguageServer/settings.ts

```diff
@@ -30,7 +31,14 @@ export class CppSettings extends Settings {
         super("C_Cpp", resource);
     }
 
-    public get clangFormatPath(): string { return super.Section.get<string>("clang_format_path"); }
+    public get clangFormatPath(): string {
+        let path: string = super.Section.get<string>("clang_format_path");
+        if (!path) {
+            path = which.sync('clang-format', {nothrow: true});
```

Comment:
Is it possible to get the bundled version without taking a dependency on `which`?

Reply:
I'm not sure I follow.  Why not use `which`?

Reply:
Bob suggested `which` specifically.  :)  https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/223355

Reply:
I was wondering if there was a way to reduce dependencies if possible. But if it's easier to use module `which`, then I agree.